source/developers/cook-books/blueprint/index.rst

```diff
@@ -152,7 +158,98 @@ On Crafter 3.0
+
+         .. code-block:: sh
+
+            git add <filename>
+
+         for each filename. Or, to add all at once use:
+
+         .. code-block:: sh
+
+	        git add --all
+
+       * And once you are done, commit them with the following command:
+
+         .. code-block:: sh
+
+            git commit -m "<the commit’s description>"
+
+       * No need to push, there’s no remote configured. You can also use any Git client. Now, it will be available when you create a new site.
+
+.. _craftercms-plugin-yaml-file:
+
+What's in the craftercms-plugin.yaml file?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``craftercms-plugin.yaml`` file contains information for use in Crafter CMS.  We'll take a look at a file used for a blueprint.  Here's a sample taken from the  ``craftercms-plugin.yaml`` for the Empty blueprint.
+
+.. code-block:: yaml
+    :linenos:
+
+    # This file describes a plugin for use in Crafter CMS
+
+    # The version of the format for this file
+    descriptorVersion: 2
+
+    # Describe the plugin
+    plugin:
+      type: blueprint
+      id: org.craftercms.blueprint.empty
+      name: Empty Blueprint
+      tags:
+        - blueprint
+        - website
+      version:
+        major: 1
+        minor: 0
+        patch: 0
+      description: |
+        Simple empty blueprint
+      website:
+        name: Empty Blueprint
+        url: https://craftercms.org
+    media:
+      screenshots:
+        - title: Home Page
+          description: Screenshot of the homepage
+          url: /studio/static-assets/images/blueprints/empty/bp_empty.png
+    developer:
+      company:
+        name: Crafter Software
+        email: info@craftersoftware.com
+        url: https://craftersoftware.com/
+    build:
+      id: c3d2a5444e6a24b5e0481d6ba87901d0b02716c8
+      date: 2019-01-23T00:00:00Z
+    license:
+      name: MIT
+      url: https://opensource.org/licenses/MIT
+    crafterCmsVersions:
+      - major: 3
+        minor: 1
+        patch: 1
+    crafterCmsEditions:
+      - community
+      - enterprise
+    searchEngine: Elasticsearch
+
+where the following fields are required:
+
+- ``descriptorVersion`` - The version of the format for this file (You can copy the value from the ``craftercms-plugin.yaml`` in one of the default blueprints under ``CRAFTER_INSTALL/data/repos/global/blueprints/`` if using a copy from the default)
+- ``plugin.type`` - ``blueprint`` for our purposes
+- ``plugin.id`` - a unique Id that is meaningful/recognizable to people who will be using the blueprint/plugin
+- ``plugin.name`` - blueprint/plugin name (For our blueprint example, it is the blueprint name shown in the **Choose Blueprint** screen of **Create Site**)
+- ``plugin.version`` - a version number for the blueprint
+- ``plugin.crafterCmsEditions`` - Crafter CMS editions that the blueprint applies to (possible values are, ``community`` and ``enterprise``)
```

Comment:
Is this really a required field?

Reply:
That's what @jdrossl said is required for now in slack, or are they not required? Thanks!

Reply:
Studio doesn't need it, I included it in the list thinking ahead but it can be omitted if this doc is specific for Studio

Reply:
@alhambrav let's omit that for now, it's not used.